docs/uglify-options.md

```diff
@@ -7,113 +7,94 @@ This task primarily delegates to [UglifyJS2][], so please consider the [UglifyJS
 
 
 ## mangle
-Type: `Boolean` `Object`  
```

Comment:
Ah, that explains the repetition from `docs/uglify-options.md` etc.

I'll revert this file then.